ATTRIBUTIONS.md

```diff
@@ -29,3 +29,10 @@ project.
 BSD License.
 
 http://llvm.org/
+
+## Transcendentals intrinsics
+
+The numerical algorithms for the transcendentals intrinsics are based on the scalar Cephes library.
+
+http://www.netlib.org/cephes/
+No License.
```

Comment:
Should we contact here the Cephes developers for granting us permission to use their algorithm? Some libraries do that and state it explicitly, some not.